### ISSUE TITLE
Create deck for submission

### ISSUE THREAD

**Issue by keepRunning** [GraphemeNFT/rarible-scaffold#40]
Template example
https://indorse-staging-bucket.s3.amazonaws.com/Carecards_Deck_-_NFT_VISION_HACK-COPYRIGHT.pdf

Single template. Need to select correct category targeting below sponsors
Build on Rarible Protocol, A Scalable NFT Art Project and Gaming & NFTs (last but third)

We should target
 Rarible, Indorse, IPFS

Need to have key points on
- Who we are
-  Our inspiration
- Our application/solution
- Our technology
- Scaling capabilities/ Vision for App/ Expansion/Monetization/Mainnet launch

**Comment by dcsan** [comment]
I can probably do a bunch of this, already have a concept deck 
and leave space for some screenshots
or is this something you feel best to do / anyone?

**Comment by dcsan** [comment]
https://docs.google.com/presentation/d/1-Vh8F_VnaL9vq9prcU6sgAcwtegP7faZTozjimucLr4/edit#slide=id.p

**Comment by tomosaigon** [comment]
> I can probably do a bunch of this, already have a concept deck
> and leave space for some screenshots
> or is this something you feel best to do / anyone?

Yes, please go for it, at least with the concept or framework or flow. I can fill in the details with screenshots and history.
